modules/logging-components-by-version.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//logging/logging-getting-started.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-components-by-version_{context}"]
+= Logging components by version
+
+For components with a version value of `master` or `main`, the current version is the latest commit to the respective branch.
+
+[options="header"]
+.Component versions
+|===
+| Release | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+| 5.9     | master   | main                  | v2.9.4 | main           | main       | v0.34.1
+| 5.8     | master   | v5.8                  | v2.9.4 | main           | main       | v0.28.1
+| 5.7     | master   | main                  | v2.9.4 | main           | main       | v0.28.1
+| 5.6     | master   | main                  | v2.9.4 | main           | main       | v0.21.0
```

Comment:
Loki is on all supported version at v3.1.0 and with the next z-streams (5.8.13 on Sept 26th, 5.9.7+5.6.24 on Oct 2nd) on 3.1.1. 5.7 is not supported anymore but it was lastly released with v2.9.8

Reply:
How can we make this page updated as per each z-stream release.

Reply:
We are already on v3.2.0 with Loki here :wave: